### PR TITLE
minor consistency update for node's well known labels

### PR DESCRIPTION
**Pull request by ksdpmx**
## What does this PR change?
* minor consistency update for node's well known labels
* minor typo fix

## Does this PR relate to any other PRs?
* no

## How will this PR impact users?
* no impact

## Does this PR address any GitHub or Zendesk issues?
*

## How was this PR tested?
* `go test`

## Does this PR require changes to documentation?
* no change

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next OpenCost release? If not, why not?
*

### CODE REVIEW

**Comment by mattray** [comment]
Thanks for the PR @ksdpmx! Any @opencost/opencost-maintainers want to weigh in on any reason not to merge this?

**Comment by mbolt35** [comment]
Thanks for the contribution @ksdpmx !

For other reviewers, here are the values for these consts: 
```
LabelInstanceType       = "beta.kubernetes.io/instance-type"
LabelInstanceTypeStable = "node.kubernetes.io/instance-type"
```

It looks like some of the test cases were updated to use the correct up to date values, but I _think_ we intended to mix legacy constants and current stable constants in order to test the compatibility layer. I think having up to date constant values is probably fine and shouldn't block this PR. Would love at least one other maintainer to verify!

**Review by mbolt35 — APPROVED** [review]
This is a great tidy-up PR - would love another reviewer just to verify the test case updates to ensure I'm not missing something.

**Review by AjayTripathy — APPROVED** [review]

**Pull request merged**
